backends/arm/passes/arm_pass_manager.py

```diff
@@ -45,3 +50,8 @@ def transform_to_backend_pipeline(
                     self.add_pass(AnnotateChannelsLastDimOrder())
 
         return self._transform(graph_module)
+
+    def transform_for_annotation_pipeline(self, graph_module: torch.fx.GraphModule):
+        self.add_pass(DecomposeDivPass())
```

Comment:
I guess the motivation for [this API](https://github.com/pytorch/pytorch/blob/183c32fd3b665414dc3700c2747da55406b14156/torch/ao/quantization/quantizer/quantizer.py#L138-L150) is clear and this is allowed, however I do want to make sure modifying graph in the Quantizer is something we are OK with from general ET lowering point of view. I also see SDPA as a valid use case.

cc @kimishpatel, @mergennachin - any comments?

Reply:
Ok - I'll resolve the merge conflict and mark this as ready for review tomorrow unless I get new comments.